graphettes/Graph.cpp

```diff
@@ -497,13 +499,16 @@ void Graph::rewireRandomEdges(double rewiredEdgesProportion) {
 }
 
 string Graph::autogenFilesFolder() {
-    return "autogenerated/" + name + "/";
+    return GRAPHETTE_AUTOGENEREATED_FILES_FOLDER;
+}
+string Graph::serializedFilesFolder() {
+    return autogenFilesFolder()+"serialized/"
 }
 
 vector<vector<uint> > Graph::loadGraphletDegreeVectors() {
     string gdvsFileName = autogenFilesFolder() + name + "_gdv.bin";
     uint n = getNumNodes();
-    if (fileExists(gdvsFileName)) {
+    if (fileExists(gdvsFileName) && !newerGraphAvailable("networks/"+name+"/"+name+".gw", gdvsFileName)) {
```

Comment:
This assumes the graph is in a file called, for example, "networks/species/species.gw". What if the command line given was "networks/species.el"? I'm not 100% this is wrong, but please check it and get back to me--it should check if whatever graph name was given on the command line is (or is not) newer than the gdvsFilename.

Note that there is a potential conflict between the two names I gave above; networks/species/species.gw, and networks/species.el. Since we are moving away from species subdirectories, hopefully this ambiguity will go away as well.

Reply:
This code is in graphettes/Graph.cpp, not src/Graph.cpp - the Graph.cpp for graphettes always looks for a gw file in `networks/"+name+"/"+name+".gw`. I assumed this was some special scenario for graphettes and left it the same.  This is the loadGraph function from graphettes/Graph.cpp:
`Graph Graph::loadGraph(string name) {
    Graph g;
    g.loadGwFile("networks/"+name+"/"+name+".gw");
    g.name = name;
    return g;
}`